### ISSUE TITLE
get_all saved filters fails due to data format conversion

### ISSUE THREAD

**Issue by dpajin-kentik** [kentik/community_sdk_python#85]
Using library installed from the main branch: 0.2.2.dev4+ge91cb23

I have tried to get all saved filters from the Kentik Org on US cluster: 

```
filters = client.saved_filters.get_all()
```

This line fails with the following error:

```
Traceback (most recent call last):
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/requests_payload/conversions.py", line 105, in convert
    return convert_func(attr)
ValueError: invalid literal for int() with base 10: 'c878'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "kt_copy_saved_filters.py", line 46, in <module>
    filters = client_from.saved_filters.get_all()
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/api_resources/saved_filters_api.py", line 16, in get_all
    return saved_filters_payload.GetAllResponse.from_json(response.text).to_saved_filters()
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/requests_payload/saved_filters_payload.py", line 75, in to_saved_filters
    return [ftr.to_saved_filter() for ftr in self]
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/requests_payload/saved_filters_payload.py", line 75, in <listcomp>
    return [ftr.to_saved_filter() for ftr in self]
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/requests_payload/saved_filters_payload.py", line 29, in to_saved_filter
    filters_obj = self._to_filters(deepcopy(self.filters))
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/requests_payload/saved_filters_payload.py", line 42, in _to_filters
    dic["filterGroups"] = [self._to_filtergroups(group) for group in dic["filterGroups"]]
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/requests_payload/saved_filters_payload.py", line 42, in <listcomp>
    dic["filterGroups"] = [self._to_filtergroups(group) for group in dic["filterGroups"]]
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/requests_payload/saved_filters_payload.py", line 46, in _to_filtergroups
    filters = [self._to_filter(ftr) for ftr in dic["filters"]]
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/requests_payload/saved_filters_payload.py", line 46, in <listcomp>
    filters = [self._to_filter(ftr) for ftr in dic["filters"]]
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/requests_payload/saved_filters_payload.py", line 57, in _to_filter
    dic["id"] = convert_or_none(dic.get("id"), ID)
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/requests_payload/conversions.py", line 115, in convert_or_none
    return convert(attr, convert_func)
  File "/home/dpajin/Documents/kentik/community_sdk_python/kentik_api_library/kentik_api/requests_payload/conversions.py", line 107, in convert
    raise DataFormatError(str(err)) from err
kentik_api.public.errors.DataFormatError: invalid literal for int() with base 10: 'c878'

```

**Comment by mmac-m3a** [comment]
Thanks for the report Dušan. We will look into it.

**Comment by dpajin-kentik** [comment]
fixed in the latest main! thanks @mmac-m3a

**Comment by mmac-m3a** [comment]
Fixed in v0.3.0
